quickwit/quickwit-search/Cargo.toml

```diff
@@ -60,3 +60,4 @@ quickwit-storage = { workspace = true, features = ["testsuite"] }
 
 [features]
 testsuite = []
+ci-test = []
```

Comment:
I don't think this feature is ever used, but some tests depends on it. We might want to enable it somewhere